addons/web/static/src/scss/form_view.scss

```diff
@@ -740,6 +740,9 @@ label.o_light_label, .o_light_label label {
             }
         }
         .o_setting_right_pane {
+            > .o_form_label {
+                font-weight: bold;
+            }
```

Comment:
We can put that as standard `.o_form_label` style -> `form_view.scss`

Reply:
@qsm-odoo , Previously in bs3.x default label tag has style 'font_weight: bold' but in current bs4 we do not have font-weight bold so if we fix this issue by considering only form view then in front-end label also have a normal font. so can we override label in common file so it should work at both sides (front-end and back-end)? do we have any common file in which we can directly override bootstrap label for all? Thanks

Reply:
The font-weight of frontend label does not need to be the same as in the backend, this is a theme choice. I am fine leaving them non-bold as bootstrap default.

Reply:
@qsm-odoo , Done the changes.